src/evse_man.h

```diff
@@ -44,9 +45,10 @@ typedef uint32_t EvseClient;
 #define EvseManager_Priority_Boost     200
 #define EvseManager_Priority_API       500
 #define EvseManager_Priority_Ohm       500
+#define EvseManager_Priority_Shaper    600
```

Comment:
I don't think you need an explicit priority, you should just use `EvseManager_Priority_Limit`, I would view this as a safety feature so should override everything. Although I guess there may be some cases where you want to allow manual override so maybe a config option to allow switching between this priority and limit depending on the specific usecase.

Reply:
Totally agree on that.

Reply:
> > Does the max power need to be a feed? Would it not be a fixed value?
> 
> not really for my use case. I've kept it dynamic just in case some countries has different energy plan depending of season, time and co. This info is already provided by my counter and I use it on other scripts so I use the mqtt feed to not have to change the value at different places.

I've falled back to a simple variable, the Mongoose Mqtt lib don't have any method to manually read a value from a topic . The max_power topic is only updated rarely if not never ( it's a retained value ),  can't  count on the mqtt callback.

Reply:
Interesting... I would expect you to get called back when you subscribe to the topic...

Reply:
not 100% sure, I'll have to retry later, I've had some bugs laying I've solved since.